browser/ui/webui/brave_rewards/rewards_page_handler.cc

```diff
@@ -212,6 +210,12 @@ RewardsPageHandler::RewardsPageHandler(
 
 RewardsPageHandler::~RewardsPageHandler() = default;
 
+void RewardsPageHandler::SetRewardsPage(
+    mojo::PendingRemote<mojom::RewardsPage> page) {
+  page_.reset();
```

Comment:
I don't actually know but do we need to call `reset` before rebinding? If so, good to know!